src/ChataUtils/ChataUtils.js

```diff
@@ -537,7 +541,14 @@ ChataUtils.openModalReport = (idRequest, options, menu, toolbar) => {
     container.classList.add('autoql-vanilla-report-problem-modal-body');
     var cancelButton = htmlToElement(
         `<div class="autoql-vanilla-chata-btn default"
-        style="padding: 5px 16px; margin: 2px 5px;">${strings.cancel}</div>`,
+        style="margin: 2px 5px;
+		padding: 10px 16px; 
```

Comment:
can we add a class instead of repeating some of these styles?

Reply:
sure I was just follow the last author who did this inline style, will change it